src/shared/components/horizontal-scroller/index.js

```diff
@@ -0,0 +1,108 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import ReactResizeDetector from 'react-resize-detector'
+import classNames from 'classnames'
+
+import ArrowButton from './arrows'
+import styles from './index.module.css'
+
+class HorizontalScroller extends PureComponent {
+  state = {
+    showSliderButtons: false,
+    isLeftArrowActive: false,
+    isRightArrowActive: true
+  }
+
+  componentDidMount () {
+    this.currentScrollX = 0
+  }
+
+  render () {
+    const { renderNavPills, className } = this.props
+    const { showSliderButtons, isLeftArrowActive, isRightArrowActive } = this.state
+    const containerClasses = classNames(styles.container, className, {
+      [styles.showButtons]: showSliderButtons
+    })
+
+    return (
+      <div className={ containerClasses }>
+        <ArrowButton className={ styles.leftButton }
```

Comment:
`styles.leftButton` and `styles.rightButton` seem to have the same content. I believe that they could be the same class